python/py-us/Portfile

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-us
+version             2.0.2
+revision            0
+
+platforms           darwin
+license             BSD
+maintainers         nomaintainer
+
+description         US: The Greatest Package in the World
+long_description    A package for easily working with US and state metadata.
+
+homepage            https://github.com/unitedstates/python-us
+
+checksums           rmd160  6b107fb52df000f6251d025484f683848d37275c \
+                    sha256  cb11ad0d43deff3a1c3690c74f0c731cff5b862c73339df2edd91133e1496fbc \
+                    size    14207
+
+python.versions     37 38
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                    port:py${python.version}-setuptools
+
+    depends_run-append \
+                    port:py${python.version}-jellyfish \
+                    port:py${python.version}-setuptools
```

Comment:
if a dependency is needed for building and at runtime, it should ideally be listed as `depends_lib` instead of listing it twice

Reply:
Thanks done.